Build failing due to d3d12TokenizedProgramFormat.hpp not found while building dxbcparser

https://github.com/microsoft/D3D9On12/actions/runs/13121201111/job/36607301553?pr=96

Cmake finds the header at configure time
```
Run cmake -B D:\a\D3D9On12\D3D9On12/build -DUSE_PIX=OFF -DCMAKE_SYSTEM_VERSION="10.0.22621.0"
-- Building for: Visual Studio 1[7](https://github.com/microsoft/D3D9On12/actions/runs/13121201111/job/36607301553?pr=96#step:4:8) 2022
-- The C compiler identification is MSVC 19.42.34436.0
-- The CXX compiler identification is MSVC 1[9](https://github.com/microsoft/D3D9On12/actions/runs/13121201111/job/36607301553?pr=96#step:4:10).42.34436.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.42.34433/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.42.34433/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test HAVE_DXCORE_LIBRARY
-- Performing Test HAVE_DXCORE_LIBRARY - Success
-- Looking for C++ include d3d[12](https://github.com/microsoft/D3D9On12/actions/runs/13121201111/job/36607301553?pr=96#step:4:13)TokenizedProgramFormat.hpp
-- Looking for C++ include d3d12TokenizedProgramFormat.hpp - found
Adding WDK-dependent project.
-- Adding DxbcSigner NuGet package...
-- Configuring done (44.9s)
-- Generating done (0.1s)
-- Build files have been written to: D:/a/D3D9On12/D3D9On12/build
```

But then the file isn't found at build time:
```
1>Checking Build System
  Building Custom Rule D:/a/D3D9On12/D3D9On12/build/_deps/directx-headers-src/CMakeLists.txt
  d3dx[12](https://github.com/microsoft/D3D9On12/actions/runs/13121201111/job/36607301553?pr=96#step:6:13)_property_format_table.cpp
  DirectX-Headers.vcxproj -> D:\a\D3D9On12\D3D9On12\build\_deps\directx-headers-build\RelWithDebInfo\DirectX-Headers.lib
  Building Custom Rule D:/a/D3D9On12/D3D9On12/build/_deps/directx-headers-src/CMakeLists.txt
  dxguids.cpp
  DirectX-Guids.vcxproj -> D:\a\D3D9On12\D3D9On12\build\_deps\directx-headers-build\RelWithDebInfo\DirectX-Guids.lib
  Building Custom Rule D:/a/D3D9On12/D3D9On12/build/_deps/d3d12translationlayer-src/DxbcParser/CMakeLists.txt
  BlobContainer.cpp
  DXBCUtils.cpp
  Generating Code...
D:\a\D3D9On12\D3D9On12\build\_deps\d3d12translationlayer-src\DxbcParser\include\dxbcutils.h(5,10): error C1083: Cannot open include file: 'd3d12TokenizedProgramFormat.hpp': No such file or directory [D:\a\D3D9On12\D3D9On12\build\_deps\d3d12translationlayer-build\DxbcParser\dxbcparser.vcxproj]
  (compiling source file '../../d3d12translationlayer-src/DxbcParser/src/BlobContainer.cpp')
  
D:\a\D3D9On12\D3D9On12\build\_deps\d3d12translationlayer-src\DxbcParser\include\dxbcutils.h(5,10): error C1083: Cannot open include file: 'd3d12TokenizedProgramFormat.hpp': No such file or directory [D:\a\D3D9On12\D3D9On12\build\_deps\d3d12translationlayer-build\DxbcParser\dxbcparser.vcxproj]
  (compiling source file '../../d3d12translationlayer-src/DxbcParser/src/DXBCUtils.cpp')
```

--- a/.github/workflows/prBuild.yml
+++ b/.github/workflows/prBuild.yml
@@ -23,10 +23,25 @@ jobs:
       with:
         nuget-version: latest
 
+    - name: Install Windows Driver Kit
+      shell: pwsh
+      run: |
+        winget install --id Microsoft.WindowsWDK.10.0.26100 `
+          --silent --accept-package-agreements --accept-source-agreements `
+          --disable-interactivity --custom "/features +"
+        $includeRoot     = "${env:ProgramFiles(x86)}\Windows Kits\10\Include\10.0.26100.0\um"
+        $requiredHeaders = @('d3d12TokenizedProgramFormat.hpp', 'd3d10umddi.h')
+        foreach ($header in $requiredHeaders) {
+          $path = Join-Path $includeRoot $header
+          if (-not (Test-Path $path)) {
+            throw "WDK install did not place $path (try /features + ?)"
+          }
+        }
+
     - name: Configure CMake
       # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
       # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
-      run: cmake -B ${{github.workspace}}/build -DUSE_PIX=OFF -DCMAKE_SYSTEM_VERSION="10.0.22621.0"
+      run: cmake -B ${{github.workspace}}/build -DUSE_PIX=OFF -DCMAKE_SYSTEM_VERSION="10.0.26100.0"
 
     - name: Nuget Restore
       run: nuget restore ${{github.workspace}}/build/D3D9on12.sln

